File: CS/WebApplication1/Helpers/ResourceDataSourceHelper.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Web;

namespace WebApplication1.Helpers {
    public class ResourceDataSourceHelper {
        public ResourceDataSourceHelper() { }

        public static List<CustomResource> GetCustomResources() {
            List<CustomResource> list = new List<CustomResource>();
            for(int i = 1; i <= 3; i++) {
                list.Add(new CustomResource() { Caption = "Resource " + i.ToString(), Color = Color.LightCoral, ResourceId = i });    
            }
            return list;
        }
    }

    public class CustomResource {
        public CustomResource() { }

        public string Caption { get; set; }
        public Color Color { get; set; }
        public int ResourceId { get; set; }
    }
}

File: VB/WebApplication1/Helpers/ResourceDataSourceHelper.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Web

Namespace WebApplication1.Helpers
	Public Class ResourceDataSourceHelper
		Public Sub New()
		End Sub

		Public Shared Function GetCustomResources() As List(Of CustomResource)
			Dim list As New List(Of CustomResource)()
			For i As Integer = 1 To 3
				list.Add(New CustomResource() With {.Caption = "Resource " & i.ToString(), .Color = Color.LightCoral, .ResourceId = i})
			Next i
			Return list
		End Function
	End Class

	Public Class CustomResource
		Public Sub New()
		End Sub

		Private privateCaption As String
		Public Property Caption() As String
			Get
				Return privateCaption
			End Get
			Set(ByVal value As String)
				privateCaption = value
			End Set
		End Property
		Private privateColor As Color
		Public Property Color() As Color
			Get
				Return privateColor
			End Get
			Set(ByVal value As Color)
				privateColor = value
			End Set
		End Property
		Private privateResourceId As Integer
		Public Property ResourceId() As Integer
			Get
				Return privateResourceId
			End Get
			Set(ByVal value As Integer)
				privateResourceId = value
			End Set
		End Property
	End Class
End Namespace